deploy/vagrant/provision/roles/ufs_gcs/files/config_alluxio.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+sed -i "s|^export ALLUXIO_UNDERFS_ADDRESS=.*|export ALLUXIO_UNDERFS_ADDRESS=gs://${GCS_BUCKET}|g" /alluxio/conf/alluxio-env.sh
```

Comment:
Good point. Relying on the script verbatim text is brittle.
One way to improve this is to refactor `alluxio-env.sh` to two parts:
1. Set all required env, including `ALLUXIO_UNDERFS_ADDRESS`, etc.
2. Set the ultimate env `ALLUXIO_JAVA_OPTS`, `ALLUXIO_MASTER_JAVA_OPTS`, `ALLUXIO_WORKER_JAVA_OPTS`, with required env values, plus optional additional env.
Hence, it would be easier for different deploy module to inject additional envs to ultimate env.
This is a major refactoring, it's better to be done in a separate PR.

Please let me know if you have simpler solution :)
Thanks!